app/views/initiatives/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class='wrapper'>
+  <h2>Edit Initiative </h2>
+  <div class='card full-width'>
+  <div class='w-form'>
+    <%= form_for @initiative do |f| %>
```

Comment:
I think there's an opportunity here to extract the form to a partial since it's used in both #new and #edit.

Reply:
We've extracted the form to be a partial